src/Examples/Flowsheets/CO2_adsorption_desorption/NETL_32D_adsorption_reactions.py

```diff
@@ -226,21 +226,26 @@ def build(self):
     def define_metadata(cls, obj):
         obj.add_properties(
             {
-                "specie_concentration": {
-                    "method": "_specie_concentration",
-                    "units": "mol/kg",
-                },
-                "rate_constant": {
+                "k_rxn": {
                     "method": "_rate_constant",
                     "units": "mol/m**3/s]",
                 },
-                "Kequil_rxn": {"method": "_Kequil_rxn", "units": None},
                 "reaction_rate": {
                     "method": "_reaction_rate",
                     "units": "mol_rxn/m**3/s",
                 },
             }
         )
+        
+        obj.define_custom_properties(
+            {
+                "specie_concentration": {
```

Comment:
Typo "specie", mainly want to make sure the name is consistent wherever it is used.

Reply:
This is from the original code, and I think I asked Chinedu about it before.